.github/workflows/_main.yaml

```
@@ -23,3 +24,12 @@ jobs:
       - uses: actions/labeler@8558fd74291d67161a8a78ce36a881fa63b766a9 # v5.0.0
         with:
           repo-token: ${{ steps.gh-app-token.outputs.token }}
+
+  ci-jobs:
+    uses: ./.github/workflows/_ci.yaml
+    permissions: {}
+
+  iac-jobs:
+    needs: labeler
+    uses: ./.github/workflows/_iac.yaml
+    permissions: {}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)